google/cloud/aiplatform/model_monitoring/alert.py

```diff
@@ -17,9 +17,16 @@
 
 from typing import Optional, List
 from google.cloud.aiplatform_v1.types import (
-    model_monitoring as gca_model_monitoring,
+    model_monitoring as gca_model_monitoring_v1,
```

Comment:
Should we use `compat` for these imports?

Reply:
`compat` by default imports from GA version of GAPIC, unless if `DEFAULT_VERSION` is set to `v1beta1`. I tried to see if the import aliases will index into the correct version by simply setting `DEFAULT_VERSION = 'v1beta1'` and then switching it back to `v1` on an ad-hoc basis, but it doesn't dynamically index in the way I was hoping for. I think it's because the symbol table isn't automatically re-written unless if we explicitly re-import. So that's why I imported both v1 and v1beta1 versions explicitly.